bin/git-conceal-unlock

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Description:
+#   A wrapper script to run `git-conceal unlock` on CI
+#   See https://github.com/Automattic/git-conceal
+#
+#   The wrapper automatically runs the `install.sh` script of `git-conceal`
+#   if it's not already installed locally before running `git-conceal unlock env:…`.
+#
+# By default, it uses the `GIT_CONCEAL_SECRET_KEY` environment variable as the key source.
+# You can override it by passing the environment variable name as the first argument.
+#
+
+set -euo pipefail
+
+env_var_name=${1:-GIT_CONCEAL_SECRET_KEY}
```

Comment:
If `GIT_CONCEAL_SECRET_KEY` is not set in the environment, the script won't fail at this stage or later when reading `env_var_name` because then we're not accessing `$GIT_CONCEAL_SECRET` but only printing `GIT_CONCEAL_SECRET`(as far as I understand).

<img width="1976" height="1692" alt="image" src="https://github.com/user-attachments/assets/373c6e00-3bd6-411a-a930-2b7a4a4e5d66" />

I suppose `git-conceal` will fail gracefully if the value given to `env:` is unset, but I wonder if we should fail early here? Or, given it's such an edge case, should we keep as is?